MANIFEST.in

```diff
@@ -2,5 +2,6 @@ include *.rst LICENSE Makefile tox.ini .coveragerc app.wsgi requirements.txt dev
 include kinto/config/kinto.tpl
 recursive-include kinto *.sql *.html *.ini *.yaml
 recursive-include docs *.rst *.png *.svg *.css *.html conf.py piwik.js
+include kinto/plugins/admin/VERSION
```

Comment:
:+1: this works because we have https://github.com/Kinto/kinto/blob/bd1f22522237798edc25de2863be8cc4acc9f77c/setup.cfg#L28